### DISCUSSION TITLE
Virtual Training Day  수강 후 PL-900 시험 신청 방법

### DISCUSSION THREAD

**Discussion by sesme100** [fusiondevkr/pl900#55] [0 공지사항]
안녕하세요 
Virtual Training Day  수강 후 PL-900 시험 신청 방법에 대해 안내 드리겠습니다. 

1. Virtual Training Day 수강 완료 하시면 수강등록 하신 이메일 (gmail.com / outlook.com등)로 수강완료 안내 이메일이 옵니다. 발신처는 PowerApps@email.microsoft.com 입니다. 하기의 캡쳐본 참고 부탁드립니다. 
2. 이메일 내용에서 "시험 세부 정보를 보려면 클릭"을 누르시면 시험예약 사이트가 나옵니다. 여기서 Pearson VUE로 예약을 누릅니다. 

3-1: Microsoft Learn에 가입이 되어 있는 경우: 클릭하자마자 자신의 Microsoft Learn 대시보드로 들어가게 됩니다.  이때 Virtual Training Day를 신청하신 이메일로 로그인이 되어있어야만 합니다.  자신의 Dashboard의 시험할인 메뉴로 들어가게 되면 100%로 할인이라는 바우처를 보실 수 있습니다. 
3-2: Microsoft Learn에 가입되어 있지 않거나 Microsoft Account가 없는 경우: 클릭하게 되면 접근 권한이 없다고 나오거나 Virtual Training Day에서 제공한 100% 할인 바우처가 보이지 않고 "할인을 확인하려면 이메일을 입력하세요"라는 안내 문구가 나옵니다. 여기에서 Virtual Training Day에 등록된 이메일을 입력합니다.  다시 한번 웹페이지를 새로고침하면 100%할인 바우처가 생성됩니다. 
또는 Microsoft Learn 사이트 대시보드로 (https://www.microsoft.com/en-us/learning/dashboard.aspx) 들어가셔셔 가입 후 PL-900 시험 예약을 하시면 100%할인 바우처를 보실 수 있습니다.  (하기 두번째 사진 참조) 
3-3: 기존 Microsoft Learn에 등록된 이메일과 Virtual Training Day에서 등록신청한 이메일이 다른 경우:  Microsoft Learn 의 Profile로 들어가셔서 이메일을 추가하시어 싱크하시면 됩니다. 

![image](https://user-images.githubusercontent.com/31263514/131344872-e1f53cde-617c-4851-95ce-71cf719236ed.png)
![image](https://user-images.githubusercontent.com/31263514/131347112-358f7c82-7131-406b-b97b-c9aca7bb795b.png)

**Comment by justinyoo** [comment]
혹시나 이 외의 방법으로 뭔가 문제가 있으시다면 여기에 계속 댓글 달아주세요!

**Comment by hcjung1225** [comment]
인증코드 입력 후 시험옵션으로 사용자 컴퓨터 선택 한 다음 시스템 테스트 창이 나오는데요.  후속 절차가 어떻게 되는지 
궁금합니다. 
사용자 컴퓨터가 노트북이고 집,회사 위치로 IP가 상이하거나 VPN 망이 있는 경우 지장이 있는지 궁금합니다.

감사합니다.
